resources/definitions/TL-D6Pro.def.json

```
@@ -0,0 +1,30 @@
+{
+    "version": 2,
+    "name": "TL-D6Pro",
+    "inherits": "tenlog_printer",
+    "metadata": {
+	"visible": true,
```

Mix of tabs and spaces
```suggestion
        "visible": true,
```